plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -203,4 +211,52 @@ public static String convertToJson(String yamlString) {
         JSONObject jsonObject=new JSONObject(map);
         return jsonObject.toString();
     }
+
+    /**
+     * Returns the JSON V7 schema for the running jenkins instance.
+     * Example Usage:
+     *      * <pre>{@code
+     *      * Schema jsonSchema = returnSchema();}
+     *      * </pre>
+     * @return Schema Returns the schema for the current jenkins instance
+     */
+    public static Schema returnSchema() {
+        JSONObject schemaObject = generateSchema();
+        JSONObject jsonSchema = new JSONObject(
+            new JSONTokener(schemaObject.toString()));
+        return SchemaLoader.load(jsonSchema);
+    }
+
+    /**
+     *Validates a given jsonObject against the schema generated for the current live jenkins instance
+     *      * Example Usage:
+     *      *      * <pre>{@code
+     *      *      * assertTrue(validateSchema(jsonSubject));}
+     *      *      * </pre>
+     * @param  jsonSubject The json Object that needs to be validated
+     * @return boolean returns true if its valid else returns false
+     */
+    public static boolean validateSchema(JSONObject jsonSubject) {
+        try {
+            returnSchema().validate(jsonSubject);
+        } catch (Exception ie) {
+            LOGGER.warning(failureMessage);
```

Comment:
should probably include the exception as well,
something like:
```suggestion
            LOGGER.log(Level.WARNING, failureMessage, ie);
```

Reply:
Done :+1: